README.md

```diff
@@ -159,6 +159,207 @@ python ./scripts/train_hf_ner.py ./data/annotations/train.spacy ./data/annotatio
+| `--model`, `-m`             | str  | GPT-3 model to use for initial predictions.                                                                                                     | `"text-davinci-003"`                |
+| `--prompt_path`, `-p`       | Path | Path to the `.jinja2` [prompt template](templates).                                                                                             | `./templates/textcat_prompt.jinja2` |
+| `--examples-path`, `-e`     | Path | Path to examples to help define the task. The file can be a .yml, .yaml or .json. If set to `None`, zero-shot learning is applied.              | `None`                              |
+| `--max-examples`, `-n`      | int  | Max number of examples to include in the prompt to OpenAI. If set to 0, zero-shot learning is always applied, even when examples are available. | 2                                   |
+| `--batch-size`, `-b`        | int  | Batch size of queries to send to the OpenAI API.                                                                                                | 10                                  |
+| `--exclusive-classes`, `-E` | bool | Flag to make the classification task exclusive.                                                                                                 | `False`                             |
+| `--verbose`, `-v`           | bool | Flag to print extra information to the terminal.                                                                                                | `False`                             |
+
+
+### Example usage
+
+The `textcat` recipes can be used for binary, multiclass, and multilabel text
+categorization. You can set this by passing the appropriate number of labels in
+the `--labels` parameter; for example, passing a single label turns it into
+binary classification and so on. We will talk about each one in the proceeding
+sections.
+
+#### Binary text categorization
+
+Suppose we want to know if a particular Reddit comment talks about a food
+recipe.  We'll send the text to GPT-3 and provide a prompt that instructs the
+predictions we want. 
+
+```
+From the text below, determine wheter or not it contains a recipe. If it is a 
+recipe, answer "accept." If it is not a recipe, answer "reject."
+
+Your answer should only be in the following format:
+answer: <string>
+reason: <string>
+
+Text:
+```
+
+For binary classification, we want GPT-3 to return "accept" if a given text is a
+food recipe and "reject" otherwise. GPT-3's suggestion is then displayed
+prominently in the UI. We can press the <kbd>ACCEPT</kbd> (check mark) button to
+include the text as a positive example or press the <kbd>REJECT</kbd> (cross
+mark) button if it is a negative example.
+
+
+```sh
+python -m prodigy textcat.openai.correct my_textcat_data data/reddit_r_cooking_sample.jsonl \
+    --labels recipe \
+    --verbose \
```

Comment:
Users are going to run this on their first "go". Do we really want to have them run in `--verbose` mode in their first try? 

Maybe. But in that case it'd be nice to add a sentence to explain the flag.

Reply:
Minor thing, but it took a while before this recipe gave me an example that's actually about a recipe. Could we maybe add this example to the top of the examples file? 

```
1/2 cup butter, 1 cup parm, and 1/2 cup heavy cream. Optionally put a half clove of minced garlic in the butter.
```

It took 18 examples before I saw my first "positive" example.